packages/menu/demo/stories/data.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const ITEMS = [
+  { value: 'plant-01', label: 'Petunia' },
+  { value: 'plant-02', label: 'Hydrangea' },
+  { value: 'separator-01', separator: true },
+  { value: 'plant-03', label: 'Violet' },
+  { value: 'plant-04', label: 'Succulent' },
+  {
+    label: 'Fruits',
```

Comment:
Can you incorporate a visible heading here with something like "Choose your favorites" so that the `checkbox` functionality is made a bit more apparent?